src/LazyOperations/AffineMap.jl

```diff
@@ -105,6 +105,7 @@ struct AffineMap{N<:Real, S<:LazySet{N}, NM, MAT<:AbstractMatrix{NM},
 end
 
 isoperationtype(::Type{<:AffineMap}) = true
+isconvextype(::Type{AffineMap{N, S, NM, MAT, VN}}) where {N, S, NM, MAT, VN} = isconvextype(S)
```

Comment:
You only need to specify the type parameters until you get to `S`. I find that easier to read, but we can keep it now.

Reply:
Ok. I'll keep this in mind.